### DISCUSSION TITLE
입퇴장 메세지와 같은 형태로 메세지 전송이 가능한가요?

### DISCUSSION THREAD

**Discussion by secneuqes** [storycraft/node-kakao#587] [Q&A]
제목 그대로 입퇴장 메세지와 같은 형태로 메세지 전송 하는 방법에 대하여 알고 싶습니다

**Comment by minmoong** [comment]
Feed 타입의 메시지를 전송하면 가능하다지만 저는 Feed 타입의 메시지 전송시 구체적으로 전달해야하는 키와 정확한 구조는 모르고 있습니다.
하지만 지금은 Feed 타입의 메시지 전송이 막혔다는 것만 알고있습니다.

**Comment by kiwioverflow** [comment]
예전에는 됬는데 지금은 안됩니다.

카카오가 막아놨습니다.
